client/state/selectors/get-site-core-update-status.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns the WordPress core update status of a site.
+ * Returns null if the site is unknown, or WordPress core hasn't been updated yet.
+ *
+ * @param  {Object}    state   Global state tree
+ * @param  {Number}    siteId  The ID of the site we're querying
+ * @return {?Boolean}          Whether WordPress core update was successful.
+ */
+export default function getSiteCoreUpdateStatus( state, siteId ) {
```

Comment:
Maybe start this with `is` or `has` to give folks a hint that it should be a boolean return value. eg `isWordpressUpdateSuccesful( state, siteId )`